README.md

```diff
@@ -30,15 +30,17 @@ Guidance around NGINX Plus is available [here](https://www.nginx.com/blog/deploy
 Dockerfiles contained in this repository are supported by and tested against NGINX Controller version 3.10 and later.
 Note: When building NGINX Plus into a container, be sure to remove repository credentials from your container.
 
-### 1.1. NGINX Controller Agent Inside Docker Container
+### 1.1. Current Limitations
```

Comment:
These are requirements of agent, I would not frame them as limitations.
The requirements of agent in a container are not different than agent with N+ in a machine.
What is different in a container environment is the possible combinations that can be dreamed up.